frame/base-fee/src/lib.rs

```diff
@@ -239,6 +242,21 @@ pub mod pallet {
 	}
 }
 
+impl<T: Config> Pallet<T> {
+	pub fn set_base_fee_per_gas_inner(value: U256) -> Weight {
```

Comment:
Can you explain the reason for doing this, it doesn't seem necessary.

Reply:
They are just some convenience functions to set the storage i.e. to update the storage values on runtime upgrade from outside the pallet or write integration tests.